src/parser/name_resolution.c

```diff
@@ -3901,6 +3901,33 @@ pt_check_unique_names (PARSER_CONTEXT * parser, const PT_NODE * p)
   return 1;			/* OK */
 }
 
+/*
+ * pt_check_contain_dot () - check if name contains DOT(.)
+ *
+ *   return:
+ *   parser(in):
+ *   p(in/out): PT_NAME node representing the name
+ *
+ * Note :
+ * This routine is used not to include DOT(.) in table, view, index,
+ * user, trigger, serial, procedure, function, or constraint names.
+ * Because DOT(.) is used as a path expression and object separator,
+ * this is to prevent difficulties in distinguishing.
+ */
+void
+pt_check_contain_dot (PARSER_CONTEXT * parser, PT_NODE * p)
+{
+  if (p != NULL)
+    {
+      const char *name = p->info.name.original;
+
+      if (name != NULL && strstr (name, ".") != NULL)
```

Comment:
It would be better to use the strchr() function instead of strstr().

Reply:
Thank you. I'll change it to the strchr() function.